lib/ZumoOta/src/App.cpp

```diff
@@ -74,45 +83,66 @@ App::~App()
 {
 }
 
-void App::start() 
+bool App::loginit()
+{
+    if (true == Logging::getInstance().registerSink(&gLogSinkSerial))
+    {
+        (void)Logging::getInstance().selectSink("Serial");
+
+        /* Set severity of logging system. */
+        Logging::getInstance().setLogLevel(CONFIG_LOG_SEVERITY);
+
+        LOG_DEBUG("LOGGER READY");
+        return true;
+    }
+    else 
+    {
+        LOG_ERROR("Fail to init Logging!");
+        return false;
+    }
+
+} 
+
+void App::start()
 {
     if (m_fileManager.init())
     {
-        Serial.println("LittleFS initialization successful");
+        LOG_DEBUG("LittleFS initialization successful");
         m_webServer.init();
     }
     else
     {
-        Serial.println("LittleFS initialization failed. The application will not start.");
+        LOG_FATAL("LittleFS initialization failed. The application will not start.");

```

Comment:
judt state the issue, not the effect. App::start is void return so I doubt the app notices.  Maybe add an App::halt() function to call after a fatal that stops in endless loop.

Reply:
It will be taken care of by the next commit.